Misc/NEWS.d/next/Library/2024-11-18-23-18-27.gh-issue-112192.DRdRgP.rst

```diff
@@ -0,0 +1 @@
+Increase the cov% in module :mod:`trace` to one decimal place.
```

Comment:
I think the NEWS entry needs some work. How can you more clearly express the change for the user?

Reply:
Look this. WDYT?
```
The :mod:`trace` module code coverage percentages with one decimal place (e.g. 20% -> 20.1%).
```
or
```
Increase the cov% in module :mod:`trace` to one decimal place (e.g. 20% -> 20.1%).
```

Reply:
How about this?

```suggestion
In the :mod:`trace` module, increase the coverage precision to one decimal.
```

Reply:
Looks like a strong sense of paragraphing. I'll modify it to your version.

Reply:
Fair enough, but I'd use double quotes for `cov%`:

```suggestion
In the :mod:`trace` module, increase the coverage precision (``cov%``) to one decimal.
```